arborist/resource_json.go

```diff
@@ -58,7 +58,8 @@ func (resourceJSON *ResourceJSON) UnmarshalJSON(data []byte) error {
 	if err != nil {
 		return err
 	}
-	resourceJSON.Path = ""
+
+	//resourceJSON.Path = ""
```

Comment:
just delete this line

Reply:
Fixed in bc25eb4ad8ccd44d86b7a24694e3989552e789c7